x-pack/plugins/ml/public/application/trained_models/models_management/test_models/models/lang_ident/lang_ident_output.tsx

```diff
@@ -6,22 +6,28 @@
  */
 
 import React, { FC } from 'react';
+import useObservable from 'react-use/lib/useObservable';
 import { i18n } from '@kbn/i18n';
 import { EuiSpacer, EuiBasicTable, EuiTitle } from '@elastic/eui';
 
-import type { FormattedLangIdentResp } from './lang_ident_inference';
+import type { LangIdentInference } from './lang_ident_inference';
 import { getLanguage } from './lang_codes';
 
 const PROBABILITY_SIG_FIGS = 3;
 
-export const LangIdentOutput: FC<{ result: FormattedLangIdentResp }> = ({ result }) => {
-  if (result.length === 0) {
+export const getLangIdentOutputComponent = (inferrer: LangIdentInference) => (
+  <LangIdentOutput inferrer={inferrer} />
+);
+
+const LangIdentOutput: FC<{ inferrer: LangIdentInference }> = ({ inferrer }) => {
```

Comment:
For the lang_ident testing, the output tab never renders for me (even with very short input strings). The results are shown in the raw output tab, but nothing in the other tab.

![image](https://user-images.githubusercontent.com/7405507/165921662-a1d64924-4040-4128-989b-9b9a4cff3fa0.png)

Reply:
Fixed in https://github.com/elastic/kibana/pull/129209/commits/6026a96f09a80ec2bea93e8512d04b6d1a4937c2